app.ts

```diff
@@ -12,80 +12,140 @@ import backendConfig from "./configs/backend-config.json";
 import backendConfigDev from "./configs/backend-config-dev.json";
 import * as yargs from "yargs";
 import {ErrorHandler} from "./ErrorHandler";
+import {MariaDAO} from "./MariaDAO";
+import * as http from "http";
+import {Connection} from "mariadb";
 
-const argv = yargs.options({
-    env: {
-        alias: 'e',
-        choices: ['dev', 'prod'],
-        demandOption: true,
-        description: 'app environment'
+
+
+function fetchConfig() {
+    const argv = yargs.options({
+        env: {
+            alias: 'e',
+            choices: ['dev', 'prod'],
+            demandOption: true,
+            description: 'app environment'
+        }
+    }).argv;
+
+    let config;
+    if (argv.env === 'prod') {
+        config = backendConfig;
+        config.env = 'prod'
```

Comment:
Kann man das `prod`  nicht in die backendConfig packen? Das selbe bei der Dev Variante?